01-Lab/SoftwareIntro.ipynb

```diff
@@ -4,253 +4,459 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Homework 1: SoftwareIntro\n",
+    "# Lab 01: SoftwareIntro\n",
     "The purpose of this homework is to guide you through opening the programs you've downloaded through Neurodebian and getting familiar with some of their basic functions.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Preparation \n",
+    "## Preparation "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download example data\n",
+    "\n",
+    "We will first download some practice MRI data from open science framework. While this is not a common outlet for scientists to share their MR imaging datasets, this is a useful repository for files that are larger than we would want to put on github and are in a simplified storage format for teaching and demonstrations like this one.\n",
+    "\n",
+    "Download from osf: https://osf.io/bprq5/\n",
```

Comment:
This can be done either in bash or python:

bash:
```
wget -O lab01_images.tar.gz https://osf.io/bprq5/download/
tar -xvf lab01_images.tar.gz
```

python:
```
import urllib.request
import tarfile

urllib.request.urlretrieve("https://osf.io/bprq5/download", "lab01_images.tar.gz")
tar = tarfile.open("lab01_images.tar.gz")
tar.extractall()
```

this makes a directory named `osfshare` that contains all the files we want.